storage/src/rocks.rs

```diff
@@ -149,38 +173,13 @@ impl RocksDb {
         Ok(Self {
             db: Some(storage),
             cf_names,
-        })
-    }
-
-    /// Opens a secondary storage instance from the given path with its given names.
-    /// If RocksDB fails to open, returns [StorageError](snarkvm_errors::storage::StorageError).
-    pub fn open_secondary_cf<P: AsRef<Path> + Clone>(
```

Comment:
I'm very happy that the secondary instance is going away with this PR; this needs to be documented in the description and the upcoming doc.